geostore/check_files_checksums/utils.py

```diff
@@ -51,7 +52,12 @@ def __init__(  # pylint:disable=too-many-arguments
 
     def log_failure(self, content: JsonObject) -> None:
         self.logger.error(
-            LOG_MESSAGE_VALIDATION_COMPLETE, extra={"outcome": Outcome.FAILED, "error": content}
+            LOG_MESSAGE_VALIDATION_COMPLETE,
+            extra={
+                "outcome": Outcome.FAILED,
+                "error": content,
+                "git_commit": get_param(ParameterName.GIT_COMMIT),
```

Comment:
The "git_commit" string is used in a bunch of places, so it should probably be in a shared file, probably `geostore/logging_keys.py`.

The same goes for the other keys ("outcome", "error", etc), so feel free to do this for all the keys as a separate PR (or file an issue to do it later).

Reply:
Added git_commit to logging_keys.py. Will create another PR for the rest of keys.